### DISCUSSION TITLE
Return void in dispatcher-Forward in controller...

### DISCUSSION THREAD

**Discussion by borisdelev** [phalcon/cphalcon#16515] [General]
I want to know something that bothers me. Its about forward in controller- by documentation i must be aware that forward is not like HTTP redirect- thats great. That is how i expect to work. But found that snippet in MVC/Controller:
 
```php
     public function saveAction()
      {
          // Forwards flow to the index action
          return $this->dispatcher->forward(
              [
                  "controller" => "people",
                  "action"     => "index",
              ]
          );
      }
```
and... forward return void... and... this is working, yes, but is somehow wrong. Must be something like that for example:
```php
     public function saveAction()
      {
          // Forwards flow to the index action
          $this->dispatcher->forward(
              [
                  "controller" => "people",
                  "action"     => "index",
              ]
          );
          return true;
      }
```

So to be clear- my question is about what is right- can we return in our action: return dispatcher->forward or make return after forward?

**Comment by niden** [comment]
The second is correct. Forward returns `void` so no "return" from a controller.

**Comment by niden** [comment]
This has been fixed https://github.com/phalcon/documentation/commit/fba58219d9d004f12a172b67e524d2a32972c06f

Thank you @borisdelev

**Comment by borisdelev** [comment]
Thank you!!!!
